spec/ddtrace/contrib/active_record/tracer_spec.rb

```diff
@@ -0,0 +1,32 @@
+require 'spec_helper'
+require 'ddtrace'
+
+require_relative 'app'
+
+RSpec.describe 'Dalli instrumentation' do
+  let(:tracer) { ::Datadog::Tracer.new(writer: FauxWriter.new) }
+
+  before(:each) do
+    Datadog.configure do |c|
+      c.use :active_record, tracer: tracer
+    end
+  end
+
+  it 'calls the instrumentation when is used standalone' do
```

Comment:
this could be considered a regression test, since undoing the patch doesn't make it pass